client/src/components/Navbar.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 
 const Navbar = () => {
-  return (
-    <header className=" viewReportButton pv3 pv3-ns ph4-m ph5-l">
-      <nav className="f6 fw6 tc">
-        <a className=" f4 link br3 ph3 white" href="/" title="Home">
-          View Reports
-        </a>
-      </nav>
-    </header>
-  );
+    return (
```

Comment:
you don't need the keyword return here cause this is an arrow function

Reply:
@Karyum  Fixed!